README.md

```diff
@@ -1,2 +1,190 @@
-# IoT-Conference-CP1
-Conference website for internet of things
+<a name="readme-top"></a>
+
+<div align="center">
+
+<!--   <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/> -->
+
+<!--   <h3><b>Microverse README Template</b></h3> -->
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 🔬 Internet of things(IoT)<a name="about-project"></a>
+
+This project is for an IoT conference web where you can see all the information about the program.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+  </ul>
+  <ul>
+    <li>CSS</li>
+  </ul>
+  <ul>
+    <li>JavaScript</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Full responsive Home and About sections.
+- Featured speakers dynamically created using JavaScript.
+- Mobile Menu functionaluity implemented in JavaScript.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://fithamlak.github.io/)
+
```

Comment:
- Hey, while you've added the live demo link, it is obvious that the link doesn't links to the current project but links to your portfolio [site](https://fithamlak.github.io/). Let's fix this issue, please.

Reply:
### MISSING VIDEO PRESENTATION [MANDATORY BUT OPTIONAL]


- Since the previous review didn't point this it, it's now an optional change, but you should add the missing video. check the project requirement to see how to add this missing requirement.